operator/README.md

```diff
@@ -1,8 +1,9 @@
 # Quobyte operator
-Deploys quobyte clients on to the nodes configured in Custom Resuource QuobyteClient.
+Deploys quobyte clients on to the nodes configured in Custom Resuource QuobyteClient.  
```

Comment:
Typo and `Definition` ist missing.

Quobyte should start with an upper case Q?

Reply:
Thank you. Corrected it.